api/src/main/java/com/github/retrooper/packetevents/wrapper/play/server/WrapperPlayServerMapData.java

```diff
@@ -113,9 +113,82 @@ public void copy(WrapperPlayServerMapData wrapper) {
         this.scale = wrapper.scale;
         this.icons = new ArrayList<>(wrapper.icons.size());
         wrapper.icons.forEach(icon -> this.icons.add(icon.clone()));
-        this.column = wrapper.column;
-        this.rows = wrapper.rows;
+        this.width = wrapper.width;
+        this.height = wrapper.height;
         this.x = wrapper.x;
         this.z = wrapper.z;
     }
+
+    public int getData() {
+        return data;
+    }
+
+    public void setData(int data) {
+        this.data = data;
+    }
+
+    public int getScale() {
+        return scale;
+    }
+
+    public void setScale(int scale) {
+        this.scale = scale;
+    }
+
+    public boolean isLocked() {
+        return locked;
+    }
+
+    public void setLocked(boolean locked) {
+        this.locked = locked;
+    }
+
+    public Collection<MapIcon> getIcons() {
+        return icons;
+    }
+
+    public void setIcons(Collection<MapIcon> icons) {
+        this.icons = icons;
+    }
+
+    @Override
```

Comment:
don't override our buffer, give it some other name. The buffer is already being used in the PacketWrapper class so this could cause issues